src/popover/popover.jsx

```diff
@@ -130,56 +131,28 @@ const Popover = React.createClass({
   },
 
   renderLayer() {
-    const {
+    let {
       animated,
-      targetOrigin,
-      className,
-      zDepth,
+      animation,
+      children,
+      style,
+      ...other,
     } = this.props;
 
-    const anchorEl = this.props.anchorEl || this.anchorEl;
-    const anchor = this.getAnchorPosition(anchorEl);
-    const horizontal = targetOrigin.horizontal.replace('middle', 'vertical');
-
-    const wrapperStyle = this.mergeAndPrefix({
-      position: 'fixed',
-      top: anchor.top,
-      left: anchor.left,
-      zIndex: this.state.muiTheme.zIndex.popover,
-      opacity: 1,
-      overflow: 'auto',
-      maxHeight: '100%',
-      transform: 'scale(0,0)',
-      transformOrigin: `${horizontal} ${targetOrigin.vertical}`,
-      transition: animated ? Transitions.easeOut('500ms', ['transform', 'opacity']) : null,
-    }, this.props.style);
-
-    const horizontalAnimation = {
-      maxHeight: '100%',
-      overflowY: 'auto',
-      transform: 'scaleX(0)',
-      opacity: 1,
-      transition: animated ? Transitions.easeOut('250ms', ['transform', 'opacity']) : null,
-      transformOrigin: `${horizontal} ${targetOrigin.vertical}`,
-    };
+    let Animation = animation;
 
-    const verticalAnimation = {
-      opacity: 1,
-      transform: 'scaleY(0)',
-      transformOrigin: `${horizontal} ${targetOrigin.vertical}`,
-      transition: animated ? Transitions.easeOut('500ms', ['transform', 'opacity']) : null,
-    };
+    if (!animated) {
+      Animation = Paper;
+      style = {position: 'fixed'};
+      if (!this.state.open) {
+        return null;
+      }
+    }
 
     return (
-      <Paper style={wrapperStyle} zDepth={zDepth} className={className} >
-        <div>
-          <div style={horizontalAnimation}>
-            <div style={verticalAnimation}>
-              {this.props.children}
-           </div>
-          </div>
-        </div>
-      </Paper>
+      <Animation {...this.props} style={style} open={this.state.open && !this.state.closing} {...other} >
```

Comment:
I think that it's cleaner like this:
`<Animation {...other} style={style} open={this.state.open && !this.state.closing}>`